tests/functional/spec/save_sign_out.spec.js

```diff
@@ -51,18 +52,12 @@ describe("SaveSignOut", () => {
     expect($(IntroThankYouPagePage.exitButton()).isExisting()).to.be.false;
   });
 
-  it("Given a logout url is not set, when I navigate the questionnaire, then I see the correct sign out buttons", () => {
-    browser.openQuestionnaire("test_introduction.json", { includeLogoutUrl: false });
+  it("Given a Census questionnaire, when I navigate the questionnaire, then I see the correct sign out buttons", () => {
```

Comment:
Updated to Census?

Reply:
Ya, previously the button text depended on us getting a log out url in the claims, which was specific to census and no longer the case any more. It is now done by survey type. It should say `complete later` for surveys where the user does not have an account, so social/census.